Eunice-Abimzy/week-6/my-react-todo/src/containers/TodosContainer.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import TodoModel from '../models/Todo'; 
+import Todos from '../components/Todos';
+import CreateTodoForm from '../components/CreateTodoForm';
+
+
+class TodosContainer extends Component  {
+  state = {
+      todos: []
+    }
+  //WHen TodosContainer mounts, fetch todo data  
+  componentDidMount(){
+    this.fetchData()
+  }
+
+  //Fetch all todos from super-crud-api
```

Comment:
Great job. Looks like you are understanding how to use models correctly. Keep it up.